NativeApp/src/components/Common/Image/styles.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+export const Image1 = styled.Image`
```

Comment:
Prefer if we just set the the image style within the components.

Reply:
i have removed image file and styled within the component